static/cms/admin/config.yml

```diff
@@ -235,5 +235,63 @@ collections:
           - { label: 'Maltese', value: 'mt' }
           - { label: 'Welsh', value: 'cy' }
           - { label: 'Belarusian', value: 'be' }
+  - name: events
```

Comment:
We ran into an issue here where netlify claimed that two collections had the same name, but for consistency it should maybe be community-events (and respective changes everwhere else). Possibly, it was just a fluke.

Reply:
Not sure why this didn't work before but it should be working now.